README.md

```diff
@@ -0,0 +1,2 @@
+# Pol_II_RvdM
```

Comment:
Also the script for intron extraction from GTF file seems to be missing, so add it and check e.g. https://github.com/jkoubele/introns-selection/blob/main/get_introns_from_gtf_file.sh (or python version: https://github.com/jkoubele/introns-selection/blob/main/get_introns_from_gtf_file.py ) for a reference.

Also be generaly careful about the different naming of UTR regions in different GTF files.